packages/core/src/file/file.element.ts

```diff
@@ -48,7 +48,7 @@ export class CdsFile extends CdsControl {
       <div>
         <cds-button size="sm" action="outline" @click="${() => this.label.click()}" ?disabled=${this.disabled}>
           <cds-icon shape="folder" aria-hidden="true"></cds-icon>
-          ${this.buttonLabel}
+          <span>${this.buttonLabel}</span>
```

Comment:
should we add an aria label here that includes the full text of the buttonLabel? In the event it gets truncated, what will happen with screen readers?

Reply:
Tested this and verified, CSS truncated text is still readable to SR. However I think we can still improve with a followup PR to split and show the file extension on the end like you pointed out earlier.